.github/workflows/Receive_PR.yml

```diff
@@ -0,0 +1,30 @@
+name: Receive_PR
+
+# read-only repo token
+# no access to secrets
+on: [push,pull_request]
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:   
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Change BASE_PATH
+        run: yq -i '.JB.BASE_PATH = "https://cgal.github.io/cgal-web"' _config.yml
+      - name: Remove demo/ (temporarily)
+        run: rm -rf demo/
```

Comment:
I add a note here: either we will have to remove those two lines, or we will need to find another way to deploy the `demo/` sub-directory.